schema-v0.0.0.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://fair.io/schema-v0.0.0.json",
+  "title": "Annual OSS Pledge Report",
+  "description": "A company's annual OSS Pledge self-report",
+  "type": "object",
+  "properties": {
+    "dateYearEnding": {
+      "description": "The end date for the year in question",
+      "type": "string",
+      "format": "date"
+    },
+    "averageNumberOfDevs": {
+      "description": "The estimated average number of software developers on staff during the year",
+      "type": "number",
+      "exclusiveMinimum" : 0
+    },
+    "payments": {
```

Comment:
Just a general question, are we asking for receipts of any kind? It would be great for the reporting to be lightweight, but having some evidence of payments whatsoever seems responsible.

Reply:
That adds a lot of complication, doesn't it? As it stands we're asking companies to host JSON files, one per year. How would we incorporate receipts?